server/svix-server/tests/e2e_endpoint.rs

```diff
@@ -507,7 +497,7 @@ async fn test_recovery_expected_retry_counts() {
         )
         .await;
 
-        sleep(Duration::from_millis(50)).await;
+        sleep(Duration::from_millis(500)).await;
```

Comment:
Why not use run_with_retries?

Reply:
The alternative is to increase number of retries and/or delay w/in `run_with_retries`, since the current fn by itself is not always enough.